packages/test-utils/src/wrapper.js

```diff
@@ -430,6 +486,9 @@ export default class Wrapper implements BaseWrapper {
    * Checks radio button or checkbox element
    */
   setChecked(checked: boolean = true): void {
+    warn(
+      `setChecked has been renamed setValue. setChecked will be removed in a future release.`
+    )
```

Comment:
Should we warn about these? There's no alternative in VTU v1, and fixing this breaking change is going to be quite painless.

Reply:
Maybe not. I'll remove them.